app/Controllers/AuthController.js

```diff
@@ -22,7 +23,10 @@ const register = catchAsync(async (req, res) => {
     await useInviteCode(req.query.inviteCode);
   }
 
-  res.send({ token, user });
+  if (validatePassword(req.body.password)) {
+    res.send({ token, user });
+  }
+  res.status(400).end();
```

Comment:
Can you do that the other way. First check if password is not valid and then `return res.status(400).end()` and then after that if do the res.send thing.